templates/commands/goldentest/test_funcs.go

```diff
@@ -80,11 +80,9 @@ func parseTestCases(ctx context.Context, location string, testNames []string) ([
 	if len(testNames) > 0 {
 		for _, testName := range testNames {
 			testPath := filepath.Join(testDir, testName)
-			_, err := os.Stat(testPath)
-			if err != nil {
-				if !common.IsStatNotExistErr(err) {
-					return nil, fmt.Errorf("failed to stat %q, %w", testPath, err)
-				}
+			if exists, err := common.Exists(testPath); err != nil {
+				return nil, fmt.Errorf("failed to stat %q, %w", testPath, err)
+			} else if !exists {
```

Comment:
Can we avoid else if here?
Can we just use 

```
if !exists {
  continue
}

Same for others.

Reply:
Sure, done.